i18n/en.json

```
@@ -187,7 +187,7 @@
     "oauth_issuer_url": "Issuer URL",
     "oauth_mobile_redirect_uri": "Mobile redirect URI",
     "oauth_mobile_redirect_uri_override": "Mobile redirect URI override",
-    "oauth_mobile_redirect_uri_override_description": "Enable when OAuth provider does not allow a mobile URI, like '{callback}'",
+    "oauth_mobile_redirect_uri_override_description": "Enable when OAuth provider does not allow a mobile URI, like \"{callback}\"",
```

If you want to preserve the single quotes in the output message, you can use two of them to escape them.

```suggestion
    "oauth_mobile_redirect_uri_override_description": "Enable when OAuth provider does not allow a mobile URI, like ''{callback}''",
```

https://formatjs.github.io/docs/core-concepts/icu-syntax#quoting--escaping

---

@Snowknight26 Applied in https://github.com/immich-app/immich/pull/16781/commits/361e591422f45cd321646432d4d1ff053eb32b4b 🙏